static-catalog/README.md

```diff
@@ -1,163 +1,215 @@
-who inherit its catalog level metadata and in turn link to other catalogs or items. A relatively small Root Catalog could
-also choose to only link to items, with no Linked Catalogs.
+A root `Catalog` should contain all the metadata that applies to all items in its catalog. It is typically where the contact information goes, as well as links to the items it contains. It usually links to one or more `catalogs`, who inherit its catalog-level metadata and in turn link to other `catalog`s or `item`s.
+A root `catalog` could also choose to include `item`s, with `catalog` links.
+Below is an example of a sub-`catalog` (that would inherit metadata from It's parent)
 
-_Linked Catalog_
+_"Child" Catalog_
 ```json
 {
-  "name": "NAIP",
-  "description": "Catalog of NAIP Imagery - 30087",
-
-  "links": [
-    { "rel": "self", "href": "naip-root.json" },
-    { "rel": "parent", "href": "naip/30087.json"}
-    { "rel": "item", "href": "30087/m_3008718_sw_16_1_20130805.json" },
-    { "rel": "item", "href": "30087/m_3008718_sw_16_1_20130806.json" },
-    ...
-  ]
+    "name": "NAIP",
+    "description": "Catalog of NAIP Imagery - 30087",
+
+    "links": [
+        { "rel": "self", "href": "https://www.fsa.usda.gov/my-real-home/naip/30087.json" },
+        { "rel": "parent", "href": "naip/30187.json"}
+        { "rel": "item", "href": "30087/m_3008718_sw_16_1_20130805.json" },
+        { "rel": "item", "href": "30087/m_3008718_sw_16_1_20130806.json" },
+        ...
+    ]
 }
 ```
 
-A 'Linked Catalog' is generally used to breakup massive numbers of catalog items in to some logical grouping, so
-that each page of links is not huge. Most commonly data is split up by date (month and day) or geography (country,
-  state). But any scheme of splitting can be used. It is a general best practice to keep the size of each linked
-  catalog under a a megabyte.
+STAC makes no formal distinction between a "root" catalog and the "child" catalogs.
+A root catalog is simply a top-most `catalog` (which has no parent).
+A nested `catalog` structure is useful (and recommended) for breaking up massive numbers of catalog items into logical groupings.
+For example, it might make sense to organize a `catalog` by date (year, month, day), or geography (continent, country, state/prov).
+Any scheme may be used, but it's considered a best practice to keep the size of each `catalog` under a a megabyte.
+
+The only required fields for a `Catalog` are the following:
+- `name`
+- `description`
+- `links`
+
+However, it is strongly recommended that a "root" catalog define metadata fields that apply to the entire `catalog` (such that child catalogs and items simply inherit these field values).
+
+The `links` section must include:
+- `self` absolute url (that points to itself)
+- `root` absolute url (that points back to the root - even if it's the root and points to itself)
```

Comment:
Not sure if we came to the conclusion that root should always be defined. I think this could make determining if the catalog is the root catalog. But I guess it could be done by an equality check between self and root.

Reply:
I think we determined the root must be defined and we took it further to identify it using an absolute URL. But I could be wrong.